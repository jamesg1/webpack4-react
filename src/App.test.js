import App from './App'
import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json';

describe('<App />', () => {
  test('should match snapshot', () => {
    const wrapper = shallow(<App />)
    expect(wrapper.find('h1').text()).toBe('Welcome to React!')
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
