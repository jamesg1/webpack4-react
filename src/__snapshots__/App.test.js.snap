// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should match snapshot 1`] = `
<div>
  <h1>
    Welcome to React!
  </h1>
</div>
`;
